Main.sublime-menu

```diff
@@ -1,5 +1,34 @@
 [
     {
+        "id": "file",
```

Comment:
I'm probably less interested in adding menu items (as I rarely use these), but I'll think about it.

Reply:
If someone specifically requests them, maybe.  But I haven't really received such a request.